Cabal/src/Distribution/Simple/Flag.hs

```diff
@@ -96,36 +101,47 @@ instance Enum a => Enum (Flag a) where
   enumFromThenTo (Flag a) (Flag b) (Flag c) = toFlag `map` enumFromThenTo a b c
   enumFromThenTo _ _ _ = []
 
+-- | Wraps a value in Flag.
```

Comment:
```suggestion
-- | Wraps a value in 'Flag'.
```

Reply:
And other instances too.

Reply:
There's a lot of missing markup, some of which was there originally. I personally intended to let that wait until the first wave died down before nitting about it.